src/server/api/routers/educations.ts

```diff
@@ -19,4 +19,68 @@ export const educationRouter = createTRPCRouter({
   getUniqueTypes: protectedProcedure.query(async () => {
     return educationService.getUniqueTypes();
   }),
+  deleteById: protectedProcedure
+    .input(z.object({ id: z.string() }))
+    .mutation(async ({ input }) => {
+      await educationService.deleteById(input.id);
+      return { success: true };
+    }),
+  create: protectedProcedure
+    .input(
+      z.object({
+        type: z.string(),
+        title: z.string(),
+        description: z.string(),
+        precondition: z.string().optional(),
+        duration: z.string().optional(),
+        lecturers: z.string().optional(),
+        courseDuration: z.string().optional(),
+        renewalDuration: z.string().optional(),
+        topics: z.string().optional(),
+      }),
+    )
+    .mutation(async ({ input }) => {
+      const result = await educationService.create(
```

Comment:
The `create` and `update` methods should receive one argument => `data: EducationFormData`

Reply:
You mean to replace `input` with data: `EducationFormData`?